src/components/Form/InputSearch.tsx

```diff
@@ -9,16 +9,16 @@ const InputSearchStyled = styled(Input.Search)`
     }
     span.ant-input-affix-wrapper {
         background: ${({ backgroundColor = colors.lightGray }) =>
-            backgroundColor};
-        box-shadow: 0px 2px 2px rgba(0, 0, 0, 0.25);
+        backgroundColor};
+        box-shadow: 0px 2px 2px ${colors.colorShadow};
         border-radius: 4px;
         &:focus-within {
-            border-color: #d9d9d9;
+            border-color: ${colors.lightBluePrimary};
```

Comment:
issue: Those color aren't the same

Reply:
using "border-color: ${colors.grayTertiary}" 

before 
![Captura de tela de 2024-10-14 21-42-43](https://github.com/user-attachments/assets/76eb3862-d177-4c7c-b175-64feee24810c)
after
![Captura de tela de 2024-10-14 21-43-19](https://github.com/user-attachments/assets/e6ade139-acb0-4678-a17d-3b3756a9d056)